src/test/java/com/thirdcc/webapp/web/rest/EventAttendeeResourceIT.java

```diff
@@ -370,17 +417,33 @@ public void getAllEventAttendees_WithEventId() throws Exception {
             .andExpect(jsonPath("$.[*].id").value(savedEventAttendee.getId().intValue()))
             .andExpect(jsonPath("$.[*].userId").value(user.getId().intValue()))
             .andExpect(jsonPath("$.[*].eventId").value(savedEvent.getId().intValue()))
-            .andExpect(jsonPath("$.[*].provideTransport").value(DEFAULT_PROVIDE_TRANSPORT.booleanValue()));
+            .andExpect(jsonPath("$.[*].provideTransport").value(DEFAULT_PROVIDE_TRANSPORT.booleanValue()))
+            .andExpect(jsonPath("$.[*].yearSession").value(DEFAULT_YEAR_SESSION));
+    }
+
+    @Test
+    @WithCurrentCCAdministrator
+    public void getAllEventAttendees_WithEventId_WithEmptyUserUniInfoDB() throws Exception {
+        // Initialize the database
+        Event savedEvent = initEventDB();
+        EventAttendee savedEventAttendee = initEventAttendeeDB();
+
+        // Get the eventAttendee
+        restEventAttendeeMockMvc.perform(get("/api/event-attendees/event/{eventId}?sort=id,desc", savedEvent.getId()))
+            .andExpect(status().isOk())
+            .andExpect(content().contentType(MediaType.APPLICATION_JSON_UTF8_VALUE))
+            .andExpect(jsonPath("$").isEmpty());
     }
 
     @Test
+    @WithCurrentCCAdministrator
     public void getAllEventAttendees_WithNonExistingEventId_ShouldThrow400() throws Exception {
         restEventAttendeeMockMvc.perform(get("/api/event-attendees/event/{eventId}?sort=id,desc", Long.MAX_VALUE))
             .andExpect(status().isBadRequest());
     }
 
     @Test
-    @WithMockUser(username = "user", roles = "USER")
+    @WithEventHead
     public void getAllEventAttendees_WithEventId_NonAdmin_ShouldThrow403() throws Exception {
```

Comment:
`getAllEventAttendees_WithEventId_NonAdmin_ShouldThrow403` 
since it is non admin, why use `@WithEventHead` instead??

Reply:
I thought Event Head is not Admin?
Admin should be the CC Administrator right?

Reply:
Resolve by https://github.com/Five-Fishes/Club-Management-Spring-Server/pull/97/commits/4533e55ee4cb81ddd20ccd03b4a792ed650b472d